src/Microsoft.DotNet.Interactive.VSCode/KernelExtensions.cs

```diff
@@ -1,20 +1,53 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.CSharp;
+using Microsoft.DotNet.Interactive.FSharp;
+using Microsoft.DotNet.Interactive.PowerShell;
 using Microsoft.DotNet.Interactive.Server;
 
 namespace Microsoft.DotNet.Interactive.VSCode
 {
     public static class KernelExtensions
     {
-        public static Task UseVSCodeHelpersAsync<TKernel>(this TKernel kernel, Kernel rootKernel) where TKernel : ISupportSetValue
+        public static TKernel UseVSCodeCommands<TKernel>(this TKernel kernel) where TKernel : Kernel
         {
-            rootKernel.RegisterCommandType<GetInput>();
+            kernel.RegisterCommandType<GetInput>();
             KernelEventEnvelope.RegisterEvent<InputProduced>();
+            return kernel;
+        }
+        public static CSharpKernel UseVSCodeHelpers(this CSharpKernel kernel)
+        {
+            var command = new SubmitCode($@"
+#r ""{typeof(VSCodeInteractiveHost).Assembly.Location.Replace("\\", "/")}""
+using static {typeof(VSCodeInteractiveHost).FullName};
+");
+            kernel.DeferCommand(command);
+            return kernel;
+        }
 
-            var interactiveHost = new VSCodeInteractiveHost(rootKernel);
-            return kernel.SetValueAsync("InteractiveHost", interactiveHost, typeof(IInteractiveHost));
+        public static FSharpKernel UseVSCodeHelpers(this FSharpKernel kernel)
+        {
+            var command = new SubmitCode($@"
+#r ""{typeof(VSCodeInteractiveHost).Assembly.Location.Replace("\\", "/")}""
+open {typeof(VSCodeInteractiveHost).FullName};
```

Comment:
Don't need the closing semicolon.

Reply:
we do that all over the place in case location contains a white space in the path I think